Tutorials/03-Wifi_Getting_Started_Guide.md

```diff
@@ -1,161 +1,153 @@
+![OpenBCI Wifi Contents](../assets/images/wifi_ganglion_what_you_need.jpg)
 
-![Wifi Pass Through Power](../assets/images/wifi_ganglion_power_2.jpg)
+#### What you need
 
-Make sure that the external power switch is set to `OFF` to **NOT** power through to the Ganglion board.
+ 1. OpenBCI WiFi Shield
+ 2. OpenBCI Ganglion Board **IMPORTANT: Must have at least v2.0.0 Ganglion firmware or newer!**
+ 3. 2 batteries, 3.7V or 4.2V LiPo or 6V AA battery will work (batteries not included)
 
 ![Wifi External Power](../assets/images/wifi_ganglion_pass_through_power.jpg)
 
-The Ganglion is not able to supply enough current to power the power-hungry wifi shield, so we put a bigger voltage regulator on the shield. In the future, the wifi shield will be able to power the Ganglion too.
+**IMPORTANT! Keep the dip switch labeled `EXT PWR` to `OFF` when using Ganglion.**
 
-## Cyton
-### What You Need
-
-![OpenBCI Wifi Contents](../assets/images/wifi_what_you_need.jpg)
+#### Important Concepts and Tutorials
 
- 1. OpenBCI Wifi Shield
- 2. OpenBCI Cyton Board
- 3. LiPo battery or 6V AA battery pack & (x4) AA batteries (batteries not included)
- 4. A computer connected to the internet
+Please read and be comfortable with the following tutorials
 
-#### 1. OpenBCI Wifi Shield
+ * [Attaching the WiFi Shield to the Ganglion](//Todo: add link)  docs.
+ * [Powering the WiFi Shield **without** powering the Ganglion](//Todo: add link)
+ * [Removing the WiFi Shield from the Ganglion](//Todo: add link)  docs.
+ 4. [Updating the WiFi Shield Firmware](http://docs.openbci.com/Hardware/12-Wifi_Programming_Tutorial)
+ * [Flashing Ganglion Firmware](http://docs.openbci.com/Hardware/09-Ganglion_Programming_Tutorial) docs.
 
-![OpenBCI Wifi Shield](../assets/images/wifi_head_shot.jpg)
+## Get the WiFi Shield On Your Wireless Network
 
-The battery connector on the back can accept 3V to 6V DC power input. The pushbutton is a reset button. For more information on the Wifi Hardware, visit the [OpenBCI Wifi](http://docs.openbci.com/Hardware/11-Wifi) page in the Hardware section. To update the shield to the latest firmware please visit the [OpenBCI Wifi Programming Tutorial](http://docs.openbci.com/Hardware/11-Wifi_Programming_Tutorial) also in the Hardware section.
+The WiFi Shield must be on the same wireless network as your computer, smart phone, or other internet connected device you wish to talk to the device with. When the WiFi Shield can't join a known network, the Shield turns into a WiFi hotspot, and will have a name such as "OpenBCI-A4AD" where the last four digits are hexadecimal and are unique to your WiFi shield. Use any wireless device to connect to launch a captive touch portal from the the WiFi Shield hotspot.
 
+The WiFi shield does not work for Enterprise level security. Use your cellphone as a hot spot or set up your own wifi network. Push The World is planning a WiFi direct connection as another alternative for researchers and those on enterprise networks.
```

Comment:
Add in front of this in bold
**NOTE: IF YOU ARE ON AN ENTERPRISE NETWORK**
or similar